Sources/BraveWallet/Crypto/Transaction Confirmations/SaferSignTransactionView.swift

```diff
@@ -0,0 +1,302 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveCore
+import BigNumber
+import Strings
+import DesignSystem
+
+struct SaferSignTransactionView: View {
+  /// The network the transaction belongs to
+  let network: BraveWallet.NetworkInfo?
+  
+  /// The address of the account making the swap
+  let fromAddress: String?
+  /// The name of the account
+  let namedFromAddress: String?
+  
+  /// The address of the recipient (applicable to CoW Swap)
+  let receiverAddress: String?
```

Comment:
if you agree, i think we should always show the receive wallet address/account

Reply:
I think fine to show for both, I'll update 👍

Reply:
Updated in [95d9bf9](https://github.com/brave/brave-ios/pull/8533/commits/95d9bf93899831106daad79bca950b1d13e83faf)